src/main/scala/sbtorgpolicies/settings/AllSettings.scala

```diff
@@ -149,7 +149,8 @@ trait AllSettings
     scalaOrganization := "org.scala-lang",
     scalaVersion := scalac.`2.12`,
     crossScalaVersions := scalac.crossScalaVersions,
-    scalacOptions ++= scalacAllOptions(scalaVersion.value)
+    scalacOptions ++= scalacAllOptions(scalaVersion.value),
+    Compile / console / scalacOptions -= "-Xlint"
```

Comment:
`-Ywarn-unused-import` should be enough no?

Reply:
I don't think so, because that's not one of the compiler flags added by the plugin. The list is here: https://github.com/47deg/sbt-org-policies/blob/66bd0de44610cf4c456cb14be0aa791f22b07851/core/src/main/scala/sbtorgpolicies/model.scala#L50-L91

Reply:
ah my bad, we should ditch `Xlint` at some point then and specify what we really need :+1:

Reply:
Yep, or switch to sbt-tpolecat

Reply:
yes even better